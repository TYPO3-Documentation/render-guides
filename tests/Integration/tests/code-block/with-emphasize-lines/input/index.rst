====================
With emphasize lines
====================

..  code-block:: python
    :emphasize-lines: 3,5

    def some_function():
        interesting = False
        print('This line is highlighted.')
        print('This one is not...')
        print('...but this one is.')
