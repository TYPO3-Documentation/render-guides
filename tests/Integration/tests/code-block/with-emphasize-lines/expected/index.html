<!-- content start -->
                <section class="section" id="with-emphasize-lines">
            <h1>With emphasize lines<a class="headerlink" href="#with-emphasize-lines" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline">Â¶</a></h1>

            <div class="code-block-wrapper code-language-python" translate="no">

        <div class="code-block-body">
            <pre class="code-block"><code class="hljs python"><span class="hljs-function"><span class="hljs-keyword">def</span> <span class="hljs-title">some_function</span><span class="hljs-params">()</span>:</span>
    interesting = <span class="hljs-literal">False</span>
<span data-emphasize-line>    print(<span class="hljs-string">'This line is highlighted.'</span>)</span>
    print(<span class="hljs-string">'This one is not...'</span>)
<span data-emphasize-line>    print(<span class="hljs-string">'...but this one is.'</span>)</span></code></pre>
            <button class="code-block-copy" title="Copy to clipboard">
                <i class="fa-regular fa-copy code-block-copy-icon"></i>
                <i class="fa-solid fa-check code-block-check-icon code-block-hide"></i>
            </button>
            <span class="code-block-check-tooltip code-block-hide" aria-live="polite">Copied!</span>
        </div>
    </div>
    </section>

        <!-- content end -->
